examples/mnist/.tensorleap/dataset.py

```diff
@@ -69,20 +68,25 @@ def calc_most_similar_class_not_gt_label_euclidean_diff(idx: int, subset: Subset
     return [min_distance, min_distance_class_label]
 
 
+####################################
+########## TL Integration ##########
+####################################
+
+
```

Comment:
I think that `_connect_to_gcs_and_return_bucket` and `_download` can be categorized as `Helpers` and take them out of `TL Integration`.